edge-agent/docker/linux/amd64/Dockerfile

```diff
@@ -1,5 +1,4 @@
 ARG base_tag=3.1.19-alpine3.14
```

Comment:
This is orthogonal to the work at hand, and it makes sense to do it as a part of a separate check-in. That said, it doesn't hurt to mention:

IIRC @damonbarry wanted to remote the base_tag from all these Dockerfiles. If you're touching them.....